### PR TITLE
 #1 : Complete administrators CRUD

### PR DESCRIPTION
**Pull request by mathieu-ducrot**
Issue : https://github.com/smartbooster/sandbox/issues/1

## Description

@nicolas-bastien here is the edit / show for the administrator CRUD.

The encoding password is base on this PR https://github.com/smartbooster/authentication-bundle/pull/2 so check it first.
And also check this PR for the isNew method https://github.com/smartbooster/sonata-bundle/pull/2

### CODE REVIEW

**Comment by mathieu-ducrot** [comment]
@nicolas-bastien launch the travis check after your review on https://github.com/smartbooster/authentication-bundle/pull/2 so it will add the admin.extension.encode_password service

File: app/config/bundles/sonata.yml [hunk 1]
```diff
@@ -17,6 +17,10 @@ sonata_admin:
     security:
         handler: sonata.admin.security.handler.noop
 
+    extensions:
```
Comment (nicolas-bastien) [inline app/config/bundles/sonata.yml]: to remove when loaded by bundle

File: src/AppBundle/Admin/User/AdministratorAdmin.php [hunk 2]
```diff
@@ -42,4 +44,39 @@ protected function configureListFields(ListMapper $list)
...
+        ;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected function configureFormFields(FormMapper $form)
+    {
+        $form
+            ->add('email', null, ['label' => 'form.label_email'])
```
Comment (nicolas-bastien) [inline src/AppBundle/Admin/User/AdministratorAdmin.php]: Can your add : ->with('fieldset.label_general', ['class' => 'col-md-7']) for email and names

File: src/AppBundle/Admin/User/AdministratorAdmin.php [hunk 3]
```diff
@@ -42,4 +44,39 @@ protected function configureListFields(ListMapper $list)
...
+            ->add('firstName', null, ['label' => 'form.label_first_name'])
+            ->add('lastName', null, ['label' => 'form.label_last_name'])
+            ->add(
```
Comment (nicolas-bastien) [inline src/AppBundle/Admin/User/AdministratorAdmin.php]: ->with('fieldset.label_password', ['class' => 'col-md-5 pull-right'])

```
<trans-unit id="fieldset.label_general">
                <source>fieldset.label_general</source>
                <target>Informations générales</target>
            </trans-unit>
            <trans-unit id="fieldset.label_password">
                <source>fieldset.label_password</source>
                <target>Mot de passe</target>
            </trans-unit>
```

**Review by nicolas-bastien — DISMISSED** [review]

**Comment by mathieu-ducrot** [comment]
@nicolas-bastien i rebase to solve the conflict on the translation file and apply what you reviewed (fieldset and removing the extension on the main config.yml)

**Pull request merged**